### ISSUE TITLE
Adding screen

### ISSUE THREAD

**Issue by anthonyaoun** [galenp/PROPERTYCOMPASS.ISSUES#113]
Hello Galen

Please see attached

AA

![propertylist-addingscreen](https://cloud.githubusercontent.com/assets/7224727/14763965/268427e8-09ea-11e6-80fd-796c4507c9ab.png)

**Comment by galenp** [comment]
Yep in short the ability to add a new listing at creation to a property list is a nice feature
